storybookjs/storybook
# Next.js: Fix for @nx/react/plugin/storybook with SVGs

Closes #



## What I did


This issue https://github.com/RyanClementsHax/storybook-addon-next/issues/100 involving storybook-addon-next and the nx plugin @nx/react/plugin/storybook was resolved  https://github.com/RyanClementsHax/storybook-addon-next/pull/101, but that fix didn't make it to this equivalent file here in @storybook/nextjs, making that issue still present when using Storybook 7 and @storybook/nextjs as the framework.

I simply copied and pasted the changes that were made there.

## How to test



1. Setup an NX monorepo with Storybook 7. Add `@nx/react/plugin/storybook` to the `main.js` config addons property.
2. Add stories of React components that contain SVGs.
3. Run `nx storybook` and ensure the operation succeeds.  


## Checklist



- [X ] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
